src/desktop/models/page.coffee

```diff
@@ -11,5 +11,8 @@ module.exports = class Page extends Backbone.Model
   urlRoot: "#{sd.API_URL}/api/v1/page"
 
   sanitizedHtml: (attr) ->
-    clean = insane(@get(attr))
+    clean = insane(@get(attr), {
+      allowedTags: ['h1', 'h2', 'div'],
+      allowedAttributes: { h1: ['style'], h2: ['style'], div: ['style'] }
+    })
```

Comment:
This seems to have broken https://staging.artsy.net/conditions-of-sale (so the test will fail?) and it should be fixed.

Reply:
This has been fixed by not specifying `allowedTags`. `h1`, `h2`, and `div` are all allowed by default.

Reply:
@sweir27 @damassi just pinging since you two touched this part of code a long time ago. I've checked the conditions of sale page looks identical to production and staging, but let me know if you see anything unexpected.